lib/ansible/config/base.yml

```
@@ -1510,6 +1510,18 @@ GALAXY_REQUIRED_VALID_SIGNATURE_COUNT:
   - The number of signatures that must be successful during GPG signature verification while installing or verifying collections.
   - This should be a positive integer or all to indicate all signatures must successfully validate the collection.
   - Prepend + to the value to fail if no valid signatures are found for the collection.
+GALAXY_RETRY_HTTP_ERROR_CODES:
+  type: list
+  default:
+    - "429" # Too many requests
+    - "520" # Galaxy rate limit error code (Cloudflare unknown error)
+    - "502" # Common error from galaxy that may represent any number of transient backend issues
```

Is it really necessary to use string literals here?
```suggestion
    - 429 # Too many requests
    - 520 # Galaxy rate limit error code (Cloudflare unknown error)
    - 502 # Common error from galaxy that may represent any number of transient backend issues
```